projects/plugins/jetpack/extensions/shared/memberships/settings.js

```diff
@@ -138,6 +178,18 @@ export function NewsletterAccessRadioButtons( {
 							{ accessLabel }
 							{ reach }
 						</label>
+						{ /*
+						 * This adds a tier selector when:
+						 * - the paid_subscribers option is selected
+						 * - stripe is selected
+						 * - there are newsletter plans
+						 * - this isn't a paywall block
```

Comment:
```suggestion
						 * - this isn't a paywall block
```
It will work if it has paywall as well.
It will be a strong incentive for user to upgrade to the next tier for example. (CC @simison for validation)

Reply:
OK, but this needs to be looked at more in depth because I believe if a user thinks they can set a paywall block they might think they could have different sections for the different subscribers but that won't work with our implementation because the tier is a post level setting. (i.e. when it changes, the entire post tier is set and not just a small section.)

I'll keep the check for now, so respond to this if we should remove it going forward.

Reply:
>  different sections for the different subscribers

I think Paid content block should be use for this use case not the Paywall

Reply:
> I'll keep the check for now, so respond to this if we should remove it going forward.

Yes, we need to remove  that check. 
The paywall will:
-  only appear to people that don't have access (not in the selected tier)
- the newsletter will be sent to everyone

Basically it will work exactly as it is now... But the access will be given to less people...